qiskit/algorithms/eigensolvers/vqd.py

```diff
@@ -115,11 +118,11 @@ def __init__(
         estimator: BaseEstimator,
         fidelity: BaseStateFidelity,
         ansatz: QuantumCircuit,
-        optimizer: Optimizer | Minimizer,
+        optimizer: Union[Optimizer, List[Optimizer]] | Union[Minimizer, List[Minimizer]],
```

Comment:
I think this should be 
```suggestion
        optimizer: Optimizer | Minimizer | Sequence[Optimizer | Minimizer],
```
to use `|` over `Union` 😄